app/assets/stylesheets/govuk_publishing_components/components/_search.scss

```diff
@@ -38,131 +40,124 @@ $large-input-size: 50px;
   border-radius: 0;
 }
 
-// IE6 + IE7 always get the simplest version, regardless of whether javascript is enabled
-@if ($is-ie == false) or ($ie-version >= 8) {
-  .js-enabled {
-    .gem-c-search__label {
-      @include core-19($line-height: $input-size, $line-height-640: $input-size);
+.js-enabled {
+  .gem-c-search__label {
+    @include govuk-font($size: 19, $line-height: $input-size);
 
-      position: absolute;
-      left: 15px;
-      top: 1px;
-      z-index: 1;
-      color: $secondary-text-colour;
-    }
+    position: absolute;
+    left: 15px;
```

Comment:
```suggestion
    left: govuk-spacing(3);
```

Reply:
Added the suggestion. I skipped it as Andy has some concerns about using the spacing scale in other places then margins and paddings.